.github/workflows/deploy-protocol-to-miannet.yaml

```diff
@@ -78,6 +78,7 @@ jobs:
           default_author: user_info
           message: '[skip ci] Deployed contracts to ${{ vars.HARDHAT_NETWORK }} Mainnet'
           fetch: --no-tags --recurse-submodules=no # Prevent failure due to submodules
+          pull: '--rebase --autostash'
```

Comment:
Are you sure about rebase? It can override git history in different way than merge

Reply:
not sure, but let's try